e2e/cypress/tests/integration/enterprise/cloud/billing/cloud_pricing_modal_spec.js

```diff
@@ -336,7 +336,7 @@ describe('Pricing modal', () => {
         cy.get('.CloudUsageModal').contains('Cloud Starter limits');
     });
 
-    it('should open team downgrade selection modal when attempting downgrading from paid to starter', () => {
+    it('should open team downgrade selection modal when attempting downgrading from enterprise trial to starter', () => {
```

Comment:
I don't think we want them to downgrade from Enterprise trial

Reply:
Ok. This is what I was missing since the existing test was asserting that they can indeed downgrade from a trial

Reply:
Made the updates